katello/katello.spec

```diff
@@ -173,6 +173,7 @@ Group: Applications/System
 Requires:       wget
 Requires:       curl
 Requires:       ruby
+Requires:       net-tools
 
```

Comment:
What about `Requires: /bin/systemctl` ?

Reply:
Good one. I also considered depending on ```/usr/bin/netstat``` rather than ```net-tools``` or rewriting it to just depend on ```/usr/bin/ss```

Reply:
Im not sure that will work:

```
[root@ssean ~]$ rpm -qf /usr/bin/netstat
net-tools-2.0-0.42.20160912git.fc26.x86_64
[root@ssean ~]$ rpm -q --provides net-tools-2.0-0.42.20160912git.fc26.x86_64
net-tools = 2.0-0.42.20160912git.fc26
net-tools(x86-64) = 2.0-0.42.20160912git.fc26
```

Whereas systemctl:

```
[root@ssean ~]$ rpm -qf /usr/bin/systemctl
systemd-233-6.fc26.x86_64
[root@ssean ~]$ rpm -q --provides systemd-233-6.fc26.x86_64
/bin/systemctl
/sbin/shutdown
config(systemd) = 233-6.fc26
libsystemd-shared-233.so()(64bit)
pkgconfig(systemd) = 233
pkgconfig(udev) = 233
syslog
system-setup-keyboard = 0.9
systemd = 233-6.fc26
systemd(x86-64) = 233-6.fc26
systemd-sysv = 206
systemd-units = 233-6.fc26

Reply:
So i'd say stick with `net-tools` unless you want to re-write it for `ss`.

Reply:
Sounds like rewriting to ```ss``` is better in the long run where Fedora is removing net-tools https://pagure.io/fesco/issue/1711